sqdtoolz/Experiments/Experimental/ExpRabi.py

```diff
@@ -98,6 +101,20 @@ def _post_process(self, data):
         if self._param_rabi_decay_time:
             self._param_rabi_decay_time.Value = 1.0 / dpkt['decay_rate']
 
+        if self._phase == 0:
+            axis = 'X'
+        elif self._phase == np.pi/2:
+            axis = 'Y'
+        elif self._phase == np.pi:
+            axis = '-X'
+        elif self._phase == 3*np.pi/2:
+            axis = '-Y'
+        else:
+            axis = None
+
+        if (not self._dont_update_values) and (axis is None):
+            assert False, 'Cannot update gate parameters if chosen phase is arbitrary (non-multiple of pi/2)'
```

Comment:
Raise ValueError instead. gives a better understanding that Value of the phase does not match that requirement.